### ISSUE TITLE
P2P Not Working With Spongeforge 1.12.2-2838-7.1.9, instead of formatting memory card item, it wipes it

### ISSUE THREAD

**Issue by tjdokas** [SpongePowered/SpongeForge#3092]
**I am currently running**

- SpongeForge version: latest stable recommended
- Forge version: 2838
- Java version: 1.8
- Operating System: hosted by ubuntu 18.04, client windows 10

Plugins (8): Minecraft, Minecraft Coder Pack, SpongeAPI, Sponge, CarrotShop, EconomyLite, LuckPerms, Nucleus

Mods (53): Minecraft, Minecraft Coder Pack, Forge Mod Loader, Minecraft Forge, SpongeAPI, Sponge, AE2 Stuff, Applied Energistics 2, BD Lib, Baubles, Chisel, CoFH Core, CoFH World, CodeChicken Lib, Ender IO, Ender IO Applied Energistics Conduits, Ender IO Base, Ender IO Conduits, Ender IO Integration with Forestry, Ender IO Integration with Tinkers' Construct, Ender IO Integration with Tinkers' Construct, Ender IO Machines, Ender IO Open Computers Conduits, Ender IO Powertools, Ender IO Refined Storage Conduits, EnderCore, Extra Planets, Extra Utilities 2, Extreme Reactors, Galacticraft Core, Galacticraft Planets, Inventory Tweaks, JourneyMap, Just Enough Items, MJRLegendsLib, MachineMuse's Modular Powersuits, Mantle, Micdoodle8 Core, Numina, OpenBlocks, OpenModsLib, OpenModsLib Core, Redstone Flux, SpongeForge, TESLA, Thaumcraft, Thaumic Energistics, The Twilight Forest, Thermal Expansion, Thermal Foundation, Tinkers' Construct, What Are We Looking At, Zero CORE

Title. P2P bus and memory card work as intended in singleplayer, but with spongeforge 1.12.2-2838-7.1.9 this issue easily reproduced, whereby you attempt to use them through the same means, only in this environment it wipes the card.

I had the same issue with forge 2768 last year. Some versions of spongeforge don't produce this issue, but after attempting ten different versions this issue persists. I suspect the issue here lies with spongeforge but am filing the issue here as this mod is the only one causing problems.

Server output returns nothing when replicating the issue.

<!-- Please provide a *full* server log (and crash-report if applicable).
https://gist.github.com/tjdokas/47d87bd41b927fa593d107c4140367fc

**Comment by phit** [comment]
this was a mod bug back then and not Sponge 

https://github.com/AppliedEnergistics/Applied-Energistics-2/issues/4123
https://github.com/SpongePowered/SpongeForge/issues/2456

> P2P bus and memory card work as intended in singleplayer

test in multiplayer without Sponge and I'm quite certain it will still happen

**Comment by fipil** [comment]
Same problem on my Forge server 1.12.2 (No sponge).

Yueh, IMO it's not a Forge/Vanilla bug. IMO is problem in the AE2 source file: appeng/items/tools/ToolMemoryCard.java. 
When I commented (so removed) the overriden method:
`public EnumActionResult onItemUse( ... )`
on the line 187, the problem has been solved. From that point the memory card works as expected, it loads configuration on click to a P2P while sneaking and it clears the memory card while sneaking and clicking to air or any else non-AE block.

Tested on both server and client. Forge: forge-1.12.2-14.23.5.2838-universal.jar
